searchhelper.user.js

```diff
@@ -1,6 +1,6 @@
 // ==UserScript==
 // @name         Neopets - Search Helper
-// @version      1.0.21
+// @version      1.0.21.1
```

Comment:
IIRC Dice prefers that the version number is not bumped as they like to increment and manage the release themselves.

Reply:
resolved